packages/url/src/filter-url-for-display.js

```diff
@@ -13,8 +13,14 @@
  * @return {string} Displayed URL.
  */
 export function filterURLForDisplay( url, maxLength = null ) {
+	if ( ! url ) {
+		return '';
+	}
+
 	// Remove protocol and www prefixes.
-	let filteredURL = url.replace( /^(?:https?:)\/\/(?:www\.)?/, '' );
```

Comment:
Again, outside of this PR, but this regex could likely use some love either way, because it doesn't correspond to the standards right now, which according to [RFC 2396](http://www.ietf.org/rfc/rfc2396.txt), Appendix A, is:

```
scheme        = alpha *( alpha | digit | "+" | "-" | "." )
```

meaning that digits, plus, minus or dots can be everywhere in the scheme, except for the first character (which has to be an alphabetic one).

Not something to fix here, but definitely something we should consider improving at some point.